praise-participants.md

```diff
@@ -0,0 +1,60 @@
+## Title
+Praise Participants
+
+## Patlet
+After an inner source contribution it's important to thank the contributor for their time and effort.
+This pattern gives guidance for a way to do so that not only effectively acknowledges the contribution but also endgenders further engagement from the contributor and others.
```

Comment:
Minor simplification of this sentence (suggestion):
This pattern gives guidance that not only effectively acknowledges the contribution but also engenders further engagement from the contributor and others.

Reply:
✅